pkgs/tools/misc/trackma/default.nix

```diff
@@ -19,33 +33,49 @@ python3.pkgs.buildPythonApplication rec {
     repo = "trackma";
     rev = "v${version}";
     sha256 = "sha256-drc39ID4WYBQ/L2py57CB5OkQNfRKNigPQW0Lp8GIMc=";
+    fetchSubmodules = true; # for anime-relations submodule
   };
 
-  nativeBuildInputs = lib.optionals withGtk [ wrapGAppsHook ];
+  nativeBuildInputs = lib.optionals withGtk [ wrapGAppsHook ]
+    ++ lib.optionals withQt [ qt5.wrapQtAppsHook ];
 
   buildInputs = lib.optionals withGtk [ glib gobject-introspection gtk3 ];
 
   propagatedBuildInputs = [ python3.pkgs.urllib3  python3.pkgs.dbus-python ]
```

Comment:
Package pygobject3 is also [required for MPRIS tracker](https://aur.archlinux.org/packages/trackma-git#comment-792449), so if no separate `withMPRIS` option, then I suggest to move dbus-python to `lib.optionals stdenv.isLinux` statement, and also add pygobject3 to this statement.